Sources/StreamChatUI/ChatMessageList/Attachments/File/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -104,7 +104,7 @@ extension ChatMessageFileAttachmentListView {
             fileNameAndSizeStack.spacing = 3
 
             mainContainerStackView.axis = .horizontal
-            mainContainerStackView.alignment = .center
+            mainContainerStackView.alignment = .top
```

Comment:
Still considering if this was a good idea. Snapshot tests fail because it also changes the positioning of titles.

Reply:
Why did we changed this? We need to be careful because it could impact some customizations of some customers

Reply:
The action image should be aligned to the top but yeah, probably not the way to go since it changes many things. Might keep the action image center aligned instead.

Reply:
I'll change it back and add a note the ticket I just added about updating the item view (does not match with latest specs in figma, errors have moved, some size issues)

Reply:
[PBE-5680](https://stream-io.atlassian.net/browse/PBE-5680)